src/actions/utils/middleware/history.js

```diff
@@ -1,9 +1,8 @@
-// @flow
-
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-/* global window */
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
```

Comment:
we still need `- /* global window */`

Reply:
whoops! I accidentally deleted it, fixed this